tests/behat/features/userforms.feature

```diff
@@ -35,17 +35,18 @@ Feature: Userforms
     And I fill in "Options[GridFieldAddNewInlineButton][2][Value]" with "2"
     And I press the "Save" button
     And I follow "My userform"
+    And I click the "Form Fields" CMS tab
 
     # Create textfields
     And I press the "Add Field" button
-    And I press the "Add Field" button
     When I fill in "Form_Fields_GridFieldEditableColumns_3_Title" with "My textfield 1"
-    When I fill in "Form_Fields_GridFieldEditableColumns_4_Title" with "My textfield 2"
     And I press the "Save" button
     When I press the "Add Page Break" button
     And I press the "Add Field" button
     And I press the "Add Field" button
-    And I fill in "Form_Fields_GridFieldEditableColumns_5_Title" with "Second Page"
+    And I press the "Add Field" button
+    And I fill in "Form_Fields_GridFieldEditableColumns_4_Title" with "Second Page"
+    When I fill in "Form_Fields_GridFieldEditableColumns_5_Title" with "My textfield 2"
```

Comment:
I honestly don't know how this test used to pass, there's an assertion below that "My textfield 2" is on the second page